src/app/ui/persmission-handler.js

```diff
@@ -41,6 +41,11 @@ export class PermissionHandler {
 
   constructor () {
     this.permissions = this._getFromLocal()
+    // remove the old permissions saved in versions < 0.9.4 since the structior has been changed
+    if (!localStorage.getItem('isNewEngine')) {
```

Comment:
perhaps having a more precise flag like `isNewEngine-<version_number>` would be better, the structure might change again in the future

Reply:
well i had that before but then remove it. I can add it if you want, i didn't want to overthink

Reply:
ok let's add it then